src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt

```diff
@@ -33,6 +34,7 @@ object ClientListener {
     private var ticksPassed: Int = 0
     val chatHistory = mutableListOf<String>()
     val actionBarHistory = mutableListOf<String>()
+    val tasks = mutableListOf<Task>()
```

Comment:
This either needs to be a concurrent data structure, or have synchronized accesses at the use sites